middleware/admin/certificate_v2.py

```diff
@@ -254,13 +255,40 @@ def certificate(self):
         return self._certificate
 
     def is_valid(self, certifier):
-        return True
+        try:
+            # IMPORTANT: for now, we only allow verifying the validity of an
+            # HSMCertificateV2ElementX509 using another HSMCertificateV2ElementX509
+            # instance as certifier. That way, we simplify the validation procedure
+            # and ensure maximum compatibility wrt the underlying library used
```

Comment:
Typo? wrt -> with?

Reply:
wrt = with respect to

you reckon I should I use the long version?

Reply:
Up to you, really. This confused me, so using the long version would probably not hurt, but the message is clear anyway.

Reply:
Ended up rephrasing it cause the word "compatibility" didn't actually reflect what I was trying to say. Let me know wdyt.